Dockerfile

```diff
@@ -293,10 +291,13 @@ ENV YARN_VERSION=1.22.4
 
 ENV NETLIFY_NODE_VERSION="12.18.0"
 
-RUN /bin/bash -c ". ~/.nvm/nvm.sh && \
+RUN /bin/bash -c "mkdir /tmp/yarn-installer && \
+         curl -o- -L https://yarnpkg.com/install.sh > \"/tmp/yarn-installer/yarn-installer.sh\" && \
+         . ~/.nvm/nvm.sh && \
          nvm install --no-progress $NETLIFY_NODE_VERSION && \
          npm install -g sm grunt-cli bower elm@$ELM_VERSION && \
-             bash /usr/local/bin/yarn-installer.sh --version $YARN_VERSION && \
+         bash /tmp/yarn-installer/yarn-installer.sh --version $YARN_VERSION && \
+         rm -rf /tmp/yarn-installer && \
```

Comment:
`yarn-installer.sh` seems to be used here as well: 

https://github.com/netlify/build-image/blob/ca811f47d4c1cbd1812d1eb6ecb0c977e86d1a1d/run-build-functions.sh#L100

Removing it would prevent executing it later?

Reply:
Oh, good catch. Maybe we shouldn't move this to its own directory, then, since it will be needed at build time. Either way, I think we can deal with that separately from this PR. I'm going to remove the `yarn-installer.sh` logic from this PR.